lib/recording/RecordingManager.js

```diff
@@ -62,7 +62,7 @@ class RecordingManager {
         }
         if (!this.recordingList[entryId]) {
             logger.info(`[${entryObj.entryId}-${entryObj.recordedEntryId}] Create new recording session`);
-            let recordingEntrySession = new RecordingEntrySession(entryObj, mp4FilesQueue);
+            let recordingEntrySession = new RecordingEntrySession(entryObj, entryServerNodeId, mp4FilesQueue);
```

Comment:
it's not a good thing to pass the entryServerNodeId in the CTOR since entryServerNodeId could change during recorded session (stop/start)